mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -63,10 +64,32 @@ mesheryctl registry generate --directory <DIRECTORY_PATH>
 		// Prerequisite check is needed - https://github.com/meshery/meshery/issues/10369
 		// TODO: Include a prerequisite check to confirm that this command IS being the executED from within a fork of the Meshery repo, and is being executed at the root of that fork.
 
+		spreadsheetIdFlag, _ := cmd.Flags().GetString("spreadsheet-id")
+		registrantDefFlag, _ := cmd.Flags().GetString("registrant-def")
+		directory, _ := cmd.Flags().GetString("directory")
+
+		if spreadsheetIdFlag == "" && registrantDefFlag == "" && directory == "" {
+			return errors.New(utils.RegistryError("[ Spreadsheet ID | Registrant Connection Definition Path | Local Directory ] isn't specified", "generate"))
```

Comment:
```suggestion
			 const errorMsg = "[ Spreadsheet ID | Registrant Connection Definition Path | Local Directory ] isn't specified\n\nUsage: mesheryctl registry generate --spreadsheet-id [Spreadsheet ID] --spreadsheet-cred $CRED\nmesheryctl registry generate --spreadsheet-id [Spreadsheet ID] --spreadsheet-cred $CRED --model \"[model-name]\"\nRun 'mesheryctl registry generate --help' to see detailed help message"
			return errors.New(utils.RegistryError(errorMsg, "generate"))
```

example in model import
```bash
/mesheryctl model import
Error: [ file | filepath | URL ] isn't specified

Usage: mesheryctl model import [ file | filePath | URL ]
Run 'mesheryctl model import --help' to see detailed help message
```